src/app/pages/StoryPage/index.test.jsx

```diff
@@ -250,6 +250,7 @@ describe('Story Page', () => {
   const appEnv = process.env.SIMORGH_APP_ENV;
   beforeEach(() => {
     process.env.SIMORGH_ICHEF_BASE_URL = 'https://ichef.test.bbci.co.uk';
+    process.env.RECOMMENDATIONS_ENDPOINT = 'http://mock-recommendations-path';
```

Comment:
To ensure you don't cause side-effects on other tests I think you need a `delete process.env.RECOMMENDATIONS_ENDPOINT;` in an `afterEach`. `process.env` vars are not automatically cleaned up between tests.

Incidently I'd check if `process.env.SIMORGH_ICHEF_BASE_URL` has an afterEach as well as that will have the same issue.

Reply:
Good spot